release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -298,8 +298,190 @@ For more information, see xref:../openshift_images/image-streams-manage.adoc#ima
 
 With {product-title} {product-version}, running `oc describe` on an image now returns os/arch and digests of each manifest.
 
-//[id="ocp-4-13-ibm-z"]
-//=== IBM Z and LinuxONE
+[id="ocp-4-13-ibm-z"]
+=== {ibmzProductName} and {linuxoneProductName}
+
+With this release, {ibmzProductName} and {linuxoneProductName} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base} KVM. For installation instructions, see the following documentation:
```

Comment:
Should IBM zSystems also have a registered symbol like IBM LinuxOne?

Reply:
Only IBM(R) LinuxONE needs the registered symbol.